.github/workflows/code_style.yaml

```diff
@@ -107,8 +107,24 @@ jobs:
             psutil pygithub pglast
           pip list
           pip list --user
+
+      # Using e375302a until 1.10.1 get released including the following PR
```

Comment:
Because pylint complains due to the change I've made on upstream (https://github.com/postgrespro/testgres/commit/e375302a114cd4df3ceed54d6526f250c44c08e7). Once it get release we can just use the pip package.